### PR TITLE
(HttpClient) Move some pieces of code to separate methods

### PR DESCRIPTION
**Pull request by artemmatiushenko1**

### CODE REVIEW

**Review by DanilByaliy — APPROVED** [review]

File: client/src/utils/httpClient.ts [hunk 1]
```diff
@@ -54,30 +57,45 @@ class CustomHttpClient implements IHttpClient {
...
+    const messages = [];
+
+    if (message) {
+      messages.push(message);
+    } else if (errors) {
+      for (const error of errors) {
+        messages.push(error.msg);
+      }
+    } else {
+      messages.push('Something went wrong! 😢');
```
Comment (KhrapkoVasyl) [inline client/src/utils/httpClient.ts]: I suggest move this message to constants

**Review by KhrapkoVasyl — CHANGES_REQUESTED** [review]

File: client/src/stores/audio-player.store.ts [hunk 2]
```diff
@@ -58,10 +59,7 @@ export class AudioPlayerStore {
   handleAudioError() {
     this.hasError = true;
     this.hasLoaded = true;
-    this.rootStore.errorHandler.handle(
-      'Something went wrong while loading audio!',
-      'warning'
-    );
+    this.rootStore.errorHandler.handle(Errors.AudioLoadError, 'warning');
```
Comment (KhrapkoVasyl) [inline client/src/stores/audio-player.store.ts:62]: Is it possible to move error types for errorHandler.handle (warning, error) to a separate enum?

Same as client/src/utils/httpClient.ts Line 98

**Review by KhrapkoVasyl — CHANGES_REQUESTED** [review]

File: client/src/stores/audio-player.store.ts (hunk 2 above)
Comment (artemmatiushenko1) [reply client/src/stores/audio-player.store.ts:62]: 'warning' or 'error' are coming from the notification library that I'm using (`TypeOptions` type), and this type is a union of string literal types, so it gives me autocompletion. So, I don't think we should create a separate enum for that
https://github.com/Bogdan-Zinovij/Ecstasify/blob/69a0a77a856f9228d11e87e5912835712f7f6894/client/src/utils/errorHandler.ts#L4

Comment (KhrapkoVasyl) [reply client/src/stores/audio-player.store.ts:62]: Ok

**Review by KhrapkoVasyl — APPROVED** [review]

**Pull request merged**